SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialWorkerConnectionTest.cpp

```diff
@@ -138,8 +138,11 @@ bool FSendReserveEntityIdsRequest::Update()
 {
 	uint32_t NumOfEntities = 1;
 	USpatialWorkerConnection* Connection = ConnectionManager->GetWorkerConnection();
-	Connection->SendReserveEntityIdsRequest(NumOfEntities);
-	Connection->Flush();
+	if (Connection)
```

Comment:
could connection be a nullptr here now?

Reply:
It always could be. I had to fix these as these connection tests cause a reading from invalid memory exception which crashes the editor, if the first test to setup the connection fails :o